CONTRIBUTING.md

```diff
@@ -69,3 +69,13 @@ To add a new documentation page, you'll have to create a Markdown file in the [d
 ## Reporting Security Issues
 
 Please see [SECURITY.md](./SECURITY.md).
+
+## Internationalizing Gutenberg Plugin
```

Comment:
As defined, I might think "Localizing" would be the preferred term here for contributors looking to submit strings.

https://www.w3.org/International/questions/qa-i18n

Reply:
Okie. I have updated the heading. :)